src/nunit-gui/Model/ITestModel.cs

```diff
@@ -73,6 +73,10 @@ public interface ITestModel : IServiceLocator, System.IDisposable
 
         #region Properties
 
+        CommandLineOptions Options { get; }
```

Comment:
No, GuiOptions is purely for the commandline. But its name probably led to your confusion - which is what bad names do. Just Options is more  generic, but then the Type is CommandLineOptions so that's probably OK.